api/metrics/src/main/java/io/opentelemetry/api/metrics/GlobalMeterProvider.java

```diff
@@ -5,29 +5,17 @@
 
 package io.opentelemetry.api.metrics;
 
-import java.util.concurrent.atomic.AtomicReference;
+import io.opentelemetry.api.metrics.internal.NoopMeterProvider;
 
-/**
- * IMPORTANT: This is a temporary class, and solution for the metrics package until it will be
- * marked as stable.
- */
-public final class GlobalMeterProvider {
-  private static final Object mutex = new Object();
-  private static final AtomicReference<MeterProvider> globalMeterProvider = new AtomicReference<>();
+/** This class is a temporary solution until metrics SDK is marked stable. */
+public class GlobalMeterProvider {
```

Comment:
this change seems unrelated to the rest of the PR. Is it needed for some reason?

Reply:
The previous implementation looked like a mistake / copy-paste with missing reason for existence.  I can remove this if it's contentious.

Reply:
nah, I don't think it's a big deal...just was puzzled when I saw it in amongst the API changes.